src/paramdef.samurai

```diff
@@ -104,6 +104,16 @@ paramdef enum mode_t {
   p_help = "XYZ for Cartesian, RTZ for spherical";
 } mode;
 
+typedef enum {
+  WIND, THERMO, WIND_THERMO
+} analysis_type_t;
```

Comment:
Actually, that is a good question.  It is not clear to me how the file 'paramsdef.samurai' is actually used.  I see references to it in the CMakeList.txt file.  It might actually be dead code. I would prefer to address the dead code issues in a separate PR.

Reply:
Sounds good to me.